.travis.yml

```diff
@@ -82,20 +74,17 @@ jobs:
     install:
     script: ./scripts/build_posix.sh
   - name: "Windows pkg"
-    stage: build
     os: windows
     language: shell
     install:
     script: ./scripts/build_windows.cmd
   - name: "Linux pkgs"
-    stage: build
     language: python
     python: 3.7
     before_install:
     install:
     script: ./scripts/build_posix.sh
   - name: "Pypi pkgs"
-    stage: build
```

Comment:
Hm, not sure I understand how it works now without explicitly setting stages. The travis page seems to look same as it did before, which makes me even more confused. Could you clarify, please?

Reply:
https://docs.travis-ci.com/user/build-stages/#naming-your-build-stages

> [...] you do not have to specify the name on every single job [...] Jobs that do not have a stage name are assigned to the previous stage name [...] if you set the stage name on the first job of each stage, the build will work as expected

![](https://imgs.xkcd.com/comics/manuals.png)

Reply:
Oh, didn't know that 🙂 Could you please add a comment about it to the code, so no one has to wonder next time?